modules/theme-tools/social-menu.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Social Menu.
+ *
+ * This feature will only be activated for themes that declare their support.
+ * This can be done by adding code similar to the following during the
+ * 'after_setup_theme' action:
+ *
+ * add_theme_support( 'jetpack-social-menu' );
+ */
+
+/**
+ * Activate the Social Menu plugin.
+ *
+ * @uses current_theme_supports()
+ */
+function jetpack_social_menu_init() {
+	// Only load our code if our theme declares support
+	if ( ! current_theme_supports( 'jetpack-social-menu' ) ) {
+		return;
+	}
+
+	/*
+	 * Social Menu description.
+	 *
+	 * Rename the social menu description.
+	 *
+	 * @module theme-tools
+	 */
+	$social_menu_description = apply_filters( 'jetpack_social_menu_description', __( 'Social Menu', 'jetpack' ) );
+
+	// Register a new menu location
+	register_nav_menus( array(
+		'jetpack-social-menu' => esc_html( $social_menu_description ),
+	) );
+
+	// Enqueue CSS
+	add_action( 'wp_enqueue_scripts', 'jetpack_social_menu_script' );
+}
+add_action( 'after_setup_theme', 'jetpack_social_menu_init', 99 );
+
+/* Function to enqueue CSS */
+function jetpack_social_menu_script() {
```

Comment:
Since this enqueues a stylesheet, should we rename the function to `jetpack_social_menu_style`, to avoid any confusion?